package.json

```diff
@@ -50,13 +52,13 @@
     "css-loader": "^0.28.7",
     "enzyme": "^3.2.0",
     "enzyme-adapter-react-16": "^1.1.1",
-    "eslint": "4.19.1",
-    "eslint-plugin-jsx-a11y": "^6.1.2",
-    "eslint-plugin-jest": "^21.5.0",
+    "eslint": "^4.19.1",
```

Comment:
Was planning on enabling linting because of all the different coding styles the app but didnt get to it